scripts/digimap-ci.coffee

```diff
@@ -477,7 +482,7 @@ module.exports = (robot) ->
 
   betaCronJob = null
 
-  robot.respond /dmci start update ([0-9]+) (.*)$/, (res) ->
+  robot.respond /dmci start update\s*([0-9]*)\s*(.*)$/, (res) ->
```

Comment:
Yes, both the days and cron are optional, so the switch from one-or-more to zero-or-more digits.